include/glow/Importer/CommonOperatorLoader.h

```diff
@@ -1230,17 +1279,28 @@ class CommonOperatorLoader : public ProtobufLoader {
                          ArgumentDictionaryTy &dict) {
     NodeValue data;
     ASSIGN_VALUE_OR_RETURN_ERR(data, getNodeValueByName(op.input(0)));
-    RETURN_ERR_IF_NOT(data.dims().size() == 1, "Data must be a 1D vector.");
+    RETURN_ERR_IF_NOT(
+        data.dims().size() == 1,
+        opErrMsg(op, strFormat("GatherRanges: Data must be a 1D vector, but "
+                               "found vector size %zu ",
+                               data.dims().size())));
 
     NodeValue ranges;
     ASSIGN_VALUE_OR_RETURN_ERR(ranges, getNodeValueByName(op.input(1)));
-    RETURN_ERR_IF_NOT(ranges.dims().size() == 3, "Ranges must be a 3D vector.");
+    RETURN_ERR_IF_NOT(
+        ranges.dims().size() == 3,
+        opErrMsg(op, strFormat("GatherRanges: Ranges must be a 3D vector, but "
+                               "found vector size %zu ",
+                               ranges.dims().size())));
     RETURN_ERR_IF_NOT(ranges.dims()[2] == 2,
-                      "Last dimension of ranges must be 2.");
+                      opErrMsg(op, strFormat("GatherRanges: Last dimension of "
+                                             "ranges must be 2, but found %lu",
+                                             ranges.dims()[2])));
```

Comment:
@vbaddi There's a legit CI issue here when we use 32 bit mode -- can you fix? Probably just need to cast.

Reply:
Done.